etl/steps/data/garden/wb/2023-07-24/food_prices_for_nutrition.countries.json

```diff
@@ -0,0 +1,188 @@
+  "DEU": "Germany",
+  "DJI": "Djibouti",
+  "DMA": "Dominica",
+  "DNK": "Denmark",
+  "DOM": "Dominican Republic",
+  "DZA": "Algeria",
+  "ECU": "Ecuador",
+  "EGY": "Egypt",
+  "ESP": "Spain",
+  "EST": "Estonia",
+  "ETH": "Ethiopia",
+  "FIN": "Finland",
+  "FJI": "Fiji",
+  "FRA": "France",
+  "GAB": "Gabon",
+  "GBR": "United Kingdom",
+  "GHA": "Ghana",
+  "GIN": "Guinea",
+  "GMB": "Gambia",
+  "GNB": "Guinea-Bissau",
+  "GNQ": "Equatorial Guinea",
+  "GRC": "Greece",
+  "GRD": "Grenada",
+  "GUY": "Guyana",
+  "HKG": "Hong Kong",
+  "HND": "Honduras",
+  "HRV": "Croatia",
+  "HTI": "Haiti",
+  "HUN": "Hungary",
+  "IDN": "Indonesia",
+  "IND": "India",
+  "IRL": "Ireland",
+  "IRN": "Iran",
+  "IRQ": "Iraq",
+  "ISL": "Iceland",
+  "ISR": "Israel",
+  "ITA": "Italy",
+  "JAM": "Jamaica",
+  "JOR": "Jordan",
+  "JPN": "Japan",
+  "KAZ": "Kazakhstan",
+  "KEN": "Kenya",
+  "KGZ": "Kyrgyzstan",
+  "KHM": "Cambodia",
+  "KNA": "Saint Kitts and Nevis",
+  "KOR": "South Korea",
+  "KWT": "Kuwait",
+  "LAO": "Laos",
+  "LBR": "Liberia",
+  "LCA": "Saint Lucia",
+  "LKA": "Sri Lanka",
+  "LSO": "Lesotho",
+  "LTU": "Lithuania",
+  "LUX": "Luxembourg",
+  "LVA": "Latvia",
+  "MAR": "Morocco",
+  "MDA": "Moldova",
+  "MDG": "Madagascar",
+  "MDV": "Maldives",
+  "MEX": "Mexico",
+  "MKD": "North Macedonia",
+  "MLI": "Mali",
+  "MLT": "Malta",
+  "MMR": "Myanmar",
+  "MNE": "Montenegro",
+  "MNG": "Mongolia",
+  "MOZ": "Mozambique",
+  "MRT": "Mauritania",
+  "MSR": "Montserrat",
+  "MUS": "Mauritius",
+  "MWI": "Malawi",
+  "MYS": "Malaysia",
+  "NAM": "Namibia",
+  "NER": "Niger",
+  "NGA": "Nigeria",
+  "NIC": "Nicaragua",
+  "NLD": "Netherlands",
+  "NOR": "Norway",
+  "NPL": "Nepal",
+  "NZL": "New Zealand",
+  "OMN": "Oman",
+  "PAK": "Pakistan",
+  "PAN": "Panama",
+  "PER": "Peru",
+  "PHL": "Philippines",
+  "POL": "Poland",
+  "PRT": "Portugal",
+  "PRY": "Paraguay",
+  "PSE": "Palestine",
+  "QAT": "Qatar",
+  "ROU": "Romania",
+  "RUS": "Russia",
+  "RWA": "Rwanda",
+  "SAU": "Saudi Arabia",
+  "SDN": "Sudan",
+  "SEN": "Senegal",
+  "SGP": "Singapore",
+  "SLE": "Sierra Leone",
+  "SLV": "El Salvador",
+  "SRB": "Serbia",
+  "STP": "Sao Tome and Principe",
+  "SUR": "Suriname",
+  "SVK": "Slovakia",
+  "SVN": "Slovenia",
+  "SWE": "Sweden",
+  "SWZ": "Eswatini",
+  "SXM": "Sint Maarten (Dutch part)",
+  "SYC": "Seychelles",
+  "TCA": "Turks and Caicos Islands",
+  "TCD": "Chad",
+  "TGO": "Togo",
+  "THA": "Thailand",
+  "TJK": "Tajikistan",
+  "TTO": "Trinidad and Tobago",
+  "TUN": "Tunisia",
+  "TUR": "Turkey",
+  "TWN": "Taiwan",
+  "TZA": "Tanzania",
+  "UGA": "Uganda",
+  "URY": "Uruguay",
+  "USA": "United States",
+  "VCT": "Saint Vincent and the Grenadines",
+  "VGB": "British Virgin Islands",
+  "VNM": "Vietnam",
+  "ZAF": "South Africa",
+  "ZMB": "Zambia",
+  "ZWE": "Zimbabwe",
+  "BON": "Bonaire (WB)",
+  "EAS": "East Asia & Pacific (WB)",
+  "ECS": "Europe & Central Asia (WB)",
+  "HIC": "High-income countries",
```

Comment:
This is currently how we name income groups:
```
    MAPPING_CLASSIFICATION = {
        "..": np.nan,  # no available classification for country-year (maybe country didn't exist yet/anymore)
        "L": "Low-income countries",
        "H": "High-income countries",
        "UM": "Upper-middle-income countries",
        "LM": "Lower-middle-income countries",
        "LM*": "Lower-middle-income countries",
    }
```
For consistency, those are the names we should use. We could consider changing them (but it would be a significant refactor, similar to the one of East Timor).